packages/components/src/Dropdown/Dropdown.tsx

```diff
@@ -228,7 +319,10 @@ function Dropdown(props: DropdownProps) {
   const trigger = buttonText && <DropdownTrigger>{buttonText}</DropdownTrigger>;
 
   const optionsList = options && (
-    <DropdownOptions>
+    <DropdownOptions
+    // optionsAlign={optionsAlign}
+    // optionsWidth={optionsWidth}
```

Comment:
Are these lines supposed to be commented out here?

Reply:
Nope! Good catch!